scripts/generate-upstream-patch.sh

```diff
@@ -2,46 +2,41 @@
 
 set -eo pipefail
 
-cleanup() {
-	rm -rf "$1"
-	echo "cleaned up $1"
-}
+ROOTDIR=$(git rev-parse --show-toplevel)
+UPSTREAM_URL_PREFIX="https://github.com/integritee-network"
 
-print_help() {
-	echo "Usage:"
-	echo "$0 [-h] [-p <tag|branch|commit-hash>] [-w <tag|branch|commit-hash>]"
-	echo ""
-	echo "without any parameter, the script will generate upstream patch for both pallets and tee-worker"
-	echo "	-h print this message"
-	echo "	-p specify the tag|branch|commit-hash for upstream pallets"
-	echo "	-w specify the tag|branch|commit-hash for upstream worker"
+function cleanup() {
+   rm -rf "$1"
+   echo "cleaned up $1"
 }
 
-check_upstream() {
-	local TARGET=$1
-	local UPSTREAM_URL="$UPSTREAM_URL_PREFIX/$TARGET"
-	if [ "$(git remote get-url upstream_$TARGET 2>/dev/null)" != "$UPSTREAM_URL" ]; then
-		git remote add upstream_$TARGET $UPSTREAM_URL
-	fi
+function usage() {
+	echo "Usage:"
+    echo "For pallets: $0 p pallets-new-commit"
+    echo "For worker : $0 w worker-new-commit"
 }
 
-# This function generates a patch for the diffs between commit-A and commit-B
+# This function generates 9 patches for the diffs between commit-A and commit-B
 # of the upstream repo, where
 # commit-A: the commit recorded in ./<TARGET_DIR>/upstream_commit
 # commit-B: the HEAD commit of upstream master or a given commit
 #
-# A patch will be generated under ./<TARGET_DIR>/upstream.patch
-generate_upstream_patch() {
-	local TARGET=$1
+# Patches will be generated under ./<TARGET_DIR>/
+function generate_worker_patch() {
+	local TARGET="worker"
 	local TARGET_DIR="$ROOTDIR/$TARGET"
+    echo "TARGET: $1"
+    echo "TARGET_DIR: $TARGET_DIR"
```

Comment:
:nail_care: the indentation is not consistent; are you planning to remove the `echo` statements later?